src/pint/models/dispersion_model.py

```diff
@@ -320,42 +320,80 @@ def __init__(self):
                 description="Dispersion measure",
             )
         )
+
+        self.add_DMX_range(None, None, dmx=0, frozen=False, index=1)
+
+        self.dm_value_funcs += [self.dmx_dm]
+        self.set_special_params(["DMX_0001", "DMXR1_0001", "DMXR2_0001"])
+        self.delay_funcs_component += [self.DMX_dispersion_delay]
+
+    def add_DMX_range(self, mjd_start, mjd_end, index=None, dmx=0, frozen=True):
+        #### Setting up the DMX title convention. If index is None, want to increment the current max DMX index by 1.
+        if index is None:
+            index = self.get_prefix_mapping_component("DMX_").max() + 1
+        i = f"{int(index):04d}"
+
+        #### Making sure start<end, flipping if not 
```

Comment:
I think it's better to raise an exception - the user probably didn't intend to put them in the wrong way around, so they probably made a calculation error. Also I don't think the comment is necessary, the code obviously does exactly that.